buildenv

```diff
@@ -1,20 +1,13 @@
-export ZOPEN_GIT_URL="https://github.com/direnv/direnv.bit"
-export ZOPEN_GIT_DEPS="comp_goport make"
+export ZOPEN_GIT_URL="https://github.com/direnv/direnv.git"
+export ZOPEN_GIT_DEPS="comp_go make coreutils"
 export ZOPEN_TYPE="GIT"
 export ZOPEN_RUNTIME_DEPS="bash"
 
 export ZOPEN_CONFIGURE="skip"
 export ZOPEN_MAKE="make"
 export ZOPEN_MAKE_MINIMAL="yes"
-export ZOPEN_CHECK="skip"
-export ZOPEN_INSTALL="zopen_install"
-
-zopen_init()
-{
-  # Go installs binaries, so create the bin dir as well
-  export GOBIN=$ZOPEN_INSTALL_DIR/bin
-  mkdir -p \$ZOPEN_INSTALL_DIR/bin
-}
+export ZOPEN_CHECK_OPTS="-i test"
```

Comment:
Thanks, I didn't think of this! Only saw that `ZOPEN_CHECK` defaults to `make check` and that that won't work.